Makefile

```diff
@@ -89,12 +89,12 @@ run-tests:
 
 docker: check-vars build
 	for arch in $(ARCHS); do \
-		docker build --build-arg=GOARCH=$$arch -t $(DOCKERIMAGE)-$$arch -f Dockerfile.scratch . ; \
+		docker build --build-arg=GOARCH=$$arch --build-arg "VERSION=$(VERSION_MAJOR_MINOR_PATCH)" -t $(DOCKERIMAGE)-$$arch -f Dockerfile.scratch . ; \
```

Comment:
@ajanikow I assume it does not matter that the first build-arg is using "=" and the second " ". For consistency it might be better to only use one variant.

Reply:
Done